README.md

```diff
@@ -1,32 +1,67 @@
-# Godot Plugin
+<p align="center">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset="./addons/rivet/images/icon-text-white.svg">
+        <img src="./addons/rivet/images/icon-text-black.svg">
+    </picture>
+</p>
+<h1 align="center">Rivet Godot Plugin</h1>
+<p align="center">
+    <a href="https://rivet.gg/discord"><img src="https://img.shields.io/discord/822914074136018994"></a>
+</p>
 
-_Godot 3 is not supported at the moment._
+---
 
-## Installation
+> [!WARNING]
+> _Godot 3 is not supported at the moment._
 
-### Method 1: Godot AssetLib
 
-1. Open AssetLib
-1. Search for "Rivet API"
-1. Open _Project > Project Settings > Plugins_ and enable _Rivet API_
-1. Open _Project > Project Settings > Autoload_ and ensure that `RivetClient` and `RivetHelper` are first on the list
+## 📦 Installation
 
-### Method 2: Clone repository
+### Using [`gd-plug`](https://github.com/imjp94/gd-plug)
 
-1. Clone this repository
-1. Open AssetLib
-1. Click "Import..."
-1. Select only the _addons_ folder and click _Install_
-1. Navigate to the _plugin-godot_ repository and select _Open_
-1. Open _Project > Project Settings > Plugins_ and enable _Rivet API_
-1. Open _Project > Project Settings > Autoload_ and ensure that `RivetClient` and `RivetHelper` are first on the list
+1. Add this line to your `plug.gd`:
+```gdscript
+plug("rivet/plugin-godot")
+```
 
-## Getting started
 
-[Getting started guide](https://rivet.gg/learn/godot)
+### Manual installation
+1. [Download the latest release](https://github.com/rivet-gg/plugin-godot/releases/latest)
+2. Unpack the `addons/rivet` folder into your `/addons` folder within the Godot project
+3. Enable this addon within the Godot settings `Project > Project Settings > Plugins`
 
-## Screenshots
+## 🚀 Getting started
 
-![Screenshot](./media/screenshot.png)
+[Head over to our Learning Center to get started with Rivet](https://rivet.gg/learn/godot).
+
+## 🧐 Features
+
+### Link your game with Rivet
+One click and it's done! Linking your game with Rivet lets you change the configuration of the deployed game server, set matchmaker options, and much more!
```

Comment:
Can we include an image in the repo that shows the screens for each of these points?

Also, as I'm writing the docs, I'm finding that it might be best to start with an approach that doesn't describe the plugin with having "endless possibilities". I might reword this as such:

```suggestion
One click and your game is connected! Linking your game with Rivet gives you one-click deployments to production, easy playtesting setup, and ways to configure your game's hosting and matchmaking.
```

(I don't think this should be the final version, I didn't figure out the last sentence the best)